src/NumberInputOne.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './Input.css'
```

Comment:
Since you imported this in your Calculator.js file, you do not need to import it here.